ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -552,8 +488,7 @@ class ServicesHomeScreen extends React.Component<Props, State> {
                   dynamicHeight={this.getHeaderHeight()}
                 />
                 {this.renderInnerContent()}
-                {this.state.isLongPressEnabled &&
-                  this.renderLongPressFooterButtons()}
+                {servicesRedesignEnabled && <ServicesHomeButton />}
```

Comment:
If we have to ship an app update before this re-design is completed we lose a feature.
We should keep the current enabling/disabling feature until the new one is full completed.

```suggestion
                {servicesRedesignEnabled ? <ServicesHomeButton />  :  this.state.isLongPressEnabled &&
                  this.renderLongPressFooterButtons()}
```
I didn't test it

Reply:
Yes it actually sounds good, after or before we will release the complete functionality we should clean up a lot of useless code...that was why i decided to already clean up the longPress handlers.

I'll revert all the edits related and protect it by a FF